readthedocs/projects/tasks.py

```diff
@@ -234,18 +234,18 @@ def sync_repo(self, environment):
         )
         version_repo = self.get_vcs_repo(environment)
         version_repo.update()
-        self.sync_versions_api(version_repo)
+        self.sync_versions_task(version_repo)
         identifier = getattr(self, 'commit', None) or self.version.identifier
         version_repo.checkout(identifier)
 
-    def sync_versions_api(self, version_repo):
+    def sync_versions_task(self, version_repo):
```

Comment:
Maybe we can call this `trigger_sync_versions` or `trigger_sync_versions_task`, to avoid having the same name for this method than for the _real_ task.